app/controllers/errors_controller.rb

```diff
@@ -0,0 +1,7 @@
+class ErrorsController < ApplicationController
+  # skip_before_action :authenticate_request
+
+  def routing_error
+    head :not_found
```

Comment:
I dont understand this very well.. This will return 404. If you don't do the match in routes.. It will also return 404. So.. whats the advantage of this?